terraform/install.sh

```diff
@@ -50,6 +50,30 @@ promptForBillingAccount() {
     exit 1;
   fi
 
+ #check for permission to link account
+  found_accounts=()
+  IFS=$'\n'
```

Comment:
please use `IFS_BAK` to remember the previous settings, so it can be effectively restored.